lib/src/messages/complex_message.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:intl_translation/src/messages/message.dart';
```

Comment:
Within a package, we can use relative imports (as long as they're not trying to reach outside the `lib/` folder). So, this could be:

```
import 'message.dart';
```